### PR TITLE
Quality Assurance

### PR DESCRIPTION
**Pull request by javatlacati**
Addeded some basic QA

### CODE REVIEW

**Comment by junichi11** [comment]
There are conflicts.
Then, as I wrote before, please don't change the format (coding style). Just write along the existing code.

File: src/com/junichi11/netbeans/modules/encoding/OpenInEncodingQueryImpl.java [hunk 1]
```diff
@@ -38,19 +40,28 @@
  * @author Tomas Zezula
  */
 @ServiceProvider(service = FileEncodingQueryImplementation.class, position = 10)
-public final class OpenInEncodingQueryImpl extends FileEncodingQueryImplementation {
```
Comment (junichi11) [inline src/com/junichi11/netbeans/modules/encoding/OpenInEncodingQueryImpl.java]: Here.

File: src/com/junichi11/netbeans/modules/encoding/OpenInEncodingQueryImpl.java [hunk 2]
```diff
@@ -38,19 +40,28 @@
...
+     */
     public static final String ENCODING = "encoding"; //NOI18N
 
     @Override
-    public Charset getEncoding(FileObject file) {
+    public Charset getEncoding(final FileObject file) {
         assert file != null;
         final Object encodingName = file.getAttribute(ENCODING);
         try {
-            return encodingName instanceof String ? Charset.forName((String) encodingName) : null;
```
Comment (junichi11) [inline src/com/junichi11/netbeans/modules/encoding/OpenInEncodingQueryImpl.java]: here.

File: src/com/junichi11/netbeans/modules/encoding/actions/SaveInEncoding.java [hunk 3]
```diff
@@ -27,78 +28,85 @@
 import org.openide.util.actions.CookieAction;
 import org.openide.windows.WindowManager;
 
-@ActionID(id = "com.junichi11.netbeans.modules.encoding.actions.SaveInEncoding", category = "File")
+/**
+ * Action that saves in the specified encoding the selected file.
+ */
+@ActionID(id = "com.junichi11.netbeans.modules.encoding.actions.SaveInEncoding",
+        category = "File")
```
Comment (junichi11) [inline src/com/junichi11/netbeans/modules/encoding/actions/SaveInEncoding.java]: here.

File: src/com/junichi11/netbeans/modules/encoding/actions/SaveInEncoding.java [hunk 4]
```diff
@@ -27,78 +28,85 @@
...
-        final DataObject dataObject = activatedNodes[0].getLookup().lookup(DataObject.class);
-        FileObject fo = dataObject.getPrimaryFile();
-        File f = FileUtil.toFile(fo);
+    protected void performAction(final Node[] activatedNodes) {
+        final DataObject dataObject = activatedNodes[0].getLookup()
+                .lookup(DataObject.class);
+        FileObject fileObject = dataObject.getPrimaryFile();
+        File f = FileUtil.toFile(fileObject);
         if (f == null) {
-            f = FileUtil.normalizeFile(new File(new File(System.getProperty("user.name")), fo.getNameExt()));
```
Comment (junichi11) [inline src/com/junichi11/netbeans/modules/encoding/actions/SaveInEncoding.java]: here

File: src/com/junichi11/netbeans/modules/encoding/actions/SaveInEncoding.java [hunk 5]
```diff
@@ -27,78 +28,85 @@
...
+     *
+     * @param activatedNodes project nodes in wich it will act
+     */
     @Override
-    protected void performAction(Node[] activatedNodes) {
-        final DataObject dataObject = activatedNodes[0].getLookup().lookup(DataObject.class);
-        FileObject fo = dataObject.getPrimaryFile();
-        File f = FileUtil.toFile(fo);
+    protected void performAction(final Node[] activatedNodes) {
+        final DataObject dataObject = activatedNodes[0].getLookup()
```
Comment (junichi11) [inline src/com/junichi11/netbeans/modules/encoding/actions/SaveInEncoding.java]: here

File: src/com/junichi11/netbeans/modules/encoding/actions/ShowEncodingAction.java [hunk 6]
```diff
@@ -56,16 +56,23 @@
  *
  * @author junichi11
  */
-@ActionID(category = "View", id = "com.junichi11.netbeans.modules.encoding.ShowEncodingAction")
```
Comment (junichi11) [inline src/com/junichi11/netbeans/modules/encoding/actions/ShowEncodingAction.java]: here

File: src/com/junichi11/netbeans/modules/encoding/actions/ShowEncodingAction.java [hunk 7]
```diff
@@ -56,16 +56,23 @@
...
+@ActionID(category = "View",
+        id = "com.junichi11.netbeans.modules.encoding.ShowEncodingAction")
 @ActionRegistration(lazy = false, displayName = "#CTL_ShowEncodingAction")
 @ActionReferences({
     @ActionReference(path = "Toolbars/Encoding", position = 0)
 })
 @Messages("CTL_ShowEncodingAction=")
-public class ShowEncodingAction extends AbstractAction implements Presenter.Toolbar {
```
Comment (junichi11) [inline src/com/junichi11/netbeans/modules/encoding/actions/ShowEncodingAction.java]: here

**Comment by junichi11** [comment]
The others is the same as well.

**Comment by javatlacati** [comment]
Tjose modifications are part of basic checkstyle. Lines linger than 80 characters can't be printed.

**Comment by junichi11** [comment]
I think it's no problem. It is hard to read.

If you want to change the coding style, at first, just ask me before you change it.

Thanks.

**Pull request closed without merging**